src/lib.rs

```diff
@@ -384,6 +384,11 @@ export_cpy!(
         }
 
         fn_py set_pwm_channels_values(channels: Vec<PwmChannel>, values: Vec<u16>) {
+            if (channels.len() != values.len()) {
+                println!("The number of values is different from the number of PWM channels.");
+                return
+            }
+
```

Comment:
why not an exception?

Reply:
The library was created to not raise exception for critical applications, from what I discussed with @RaulTrombin some time ago. But I have no strong feelings about it.

Reply:
```Rust
        fn_py set_pwm_channels_values(channels: Vec<PwmChannel>, values: Vec<u16>) -> PyResult<()> {
            if (channels.len() != values.len()) {
                Err(PyValueError::new_err("The number of values is different from the number of PWM channels."))
            }
        }
```

Reply:
That's great, thanks!

Reply:
A panic?

on the navigator-rs library we used something like:
warn!("Invalid value. Value must be greater than {min_prescale}.");
and kept the things working as it's

A would avoid to panic on this, imagining the user case:
I deployed a ROV tracker solution on python:
it tracks QR code, then move the motors.

if something goes wrong during the process, panic will simply crash the entire application.

Reply:
Well, I just tested it here and it raises. So.. Should we allow raise and panic on the library ?

Reply:
I'm also inclined to not raise/panic in the library as well. We can change it later and move the code to use panic/raise.

Reply:
I don't think we want to `panic` on the Rust side, but the error handling in Python is usually idiomatically by means of exceptions (which I'm not a big fan of, but this is how things are). On the Cpp side, there's a broader range of what's idiomatic, right?